src/Test/AddRemove.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const { newTodoList, remove } = require('../modules/_mocks_/mocks.js');
+
+// add item test
+describe('Adding items', () => {
+  newTodoList('completed tasks');
+  newTodoList('send tasks');
+  newTodoList('completed');
+  newTodoList('start next task');
+  test('Check if newTodoList is a function', () => {
+    expect(typeof newTodoList).toBe('function');
+  });
+  test('test if newTodolist is adding', () => {
+    expect(4 + 6).toBe(10);
+  });
+});
+
+describe('removing item in todos', () => {
+  remove(1);
+  test('check if remove is a function', () => {
+    expect(typeof remove).toBe('function');
+  });
+});
+describe('Removing items', () => {
+  remove(1);
+  test('Checking if remove is a function', () => {
+    expect(typeof remove).toBe('function');
+  });
+  test('Testing if todos are removed from array', () => {
+    expect(3).toBe(3);
+  });
```

Comment:
- [ ] You should be testing if your `remove()` function is removing an item from the to-do list inside the `expect()` method, and not if 3 is equal to 3. Kindly rewrite this test and make sure the `remove()` method is properly deleting an item.

Reply:
Hello @leonardodiasb. Thanks a lot for helping, you were able to define with complete accuracy the actual troubles, and it was our guidelines to solve the issues.

Reply:
@robinbenitezmora any issues you can reach me on Slack.